packages/features/form-builder/FormBuilder.tsx

```diff
@@ -368,7 +368,7 @@ export const FormBuilder = function FormBuilder({
             fieldIndex: -1,
           })
         }>
-        <DialogContent>
+        <DialogContent enableOverflow>
```

Comment:
@G3root Why would we not want this to be enabled always?

Reply:
i think enabling this by default broke some modals 🤔 
cc: @Udit-takkar

Reply:
Even if that is the case, I think we should disable it if needed. But this behaviour is something that should be enabled by default.